tests/lib/rules/typescript/prefer-pascal-case-enums.test.js

```diff
@@ -8,7 +8,6 @@ const ruleTester = new RuleTester({
 
 function errorWithName(name) {
   return {
-    type: 'Identifier',
```

Comment:
I had to remove this check because when the key is a string, the type of the error is `Literal`

Reply:
That makes sense